content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,173 +2,304 @@
-the :guilabel:`Regenerate Work Entries` button, and entering the corresponding information for each
-employee.
+To print payslips, first view the individual payslips by clicking the :guilabel:`Payslips` smart
+button on the batch form. Next, select the payslips to print in the payslip list. Click the box next
+to each payslip to print, or click the box next to :guilabel:`Reference` to select all the payslips
+in the list at once. Click the :guilabel:`Print` button, and a PDF file is created with all the
+specified payslips.
 
-.. image:: work_entries/regenerate-employee.png
+.. image:: work_entries/print-payslips.png
    :align: center
-   :alt: Regenerate Work Entries button on the Work Entries Regeneration form.
+   :alt: Print button for printing the payslips.
 
-Generating payslips
-===================
+.. note::
+   The :guilabel:`Print` button does not appear until at least one payslip is selected in the list.
 
-To generate payslips, navigate to the time period the payslips should be generated for, either day,
-week, or month. When the desired pay period is displayed, click the :guilabel:`Generate Payslips`
-button.
+Time off to report
+==================
 
-.. image:: work_entries/generate-payslips.png
+If a time off request is submitted for a time period that was already processed on a payslip, the
+time off request appears in the :guilabel:`Time Off to Report`, with a status of :guilabel:`To defer
+to next payslip`. This is because the employee was already paid for that day, and it was logged as
+time spent at work, as a typical work day.
+
+In order to keep the employee's time off balances correct, the time off request must be applied to
+the following pay period. This not only ensures time off request balances are current, but it also
+eliminates the need to redo work entries, cancel paychecks, and reissue paychecks.
+
+The most common scenario when this situation occurs, is when payslips are processed a day or two
+before the pay period ends, and an employee is unexpectedly sick on one of the last days of the pay
+period. The employee puts in a time off request for a day that was already processed on a payslip as
+a regular work day. Instead of canceling the payslip, modifying the work entries, and reissuing the
+paycheck, Odoo allows for those time off requests to be applied to the following pay period instead.
+
+To view all the time off requests that need to be deferred to the next payslip, navigate to
+:menuselection:`Payroll app --> Work Entries --> Time Off to Report`. The default filter for this
+report is :guilabel:`To Defer`.
+
+All time off requests that need to be applied to the following pay period appear with a
+:guilabel:`Payslip State` of :guilabel:`To defer to next payslip`.
+
+.. image:: work_entries/time-off-to-report.png
    :align: center
-   :alt: Generate Payslips button on the Work Entry dashboard.
+   :alt: A list of all time off requests that were not approved before payslips were generated.
```

Comment:
it might be my comp, but this screenshot appears to be a *little* on the fuzzy side :) you may want to double-check and consider a fresh one during your revisions.

Reply:
Agreed- that's what happens when I take too wide a shot! I cropped out the parts I didn't need and it's much more clear now.